orbs/pulumi.yml

```diff
@@ -90,6 +90,34 @@ commands:
           name: "pulumi stack rm << parameters.stack >>"
           command: pulumi stack rm << parameters.stack >> <<# parameters.force >>--force<</ parameters.force >> --cwd << parameters.working_directory >>
 
+  # stack output command
+  stack_output:
+    parameters:
+      stack:
+        type: string
+        description: "Name of the stack to operate on."
+      property_name:
+        type: string
+        description: "Property name of the stack output to set to the environment variable."
+      env_var:
+        type: string
+        description: "Name of the environment variable to set the output property's value to."
+      log_to_stderr:
+        type: boolean
+        description: "Log to stderr instead of to files. Default true."
+        default: true
```

Comment:
I thought making this `true` by default for the CI is recommended. Am I correct?

Reply:
Nope, we shouldn't be setting that at all.

If `pulumi` writes an error to STDERR (e.g. there is a problem retrieving the output property) then ideally we would have that printed to the console and _not_ captured in the environment variable.

`$TEST_WEBSITE_URL` having a value of `"Error[404]: stack not found"` would probably be easier to track down than `$TEST_WEBSITE_URL` having no value at all, and the CircleCI error logs containing the `pulumi` output.

But either way, the output (STDOUT and STDERR) by default contains all the information `pulumi` intends to print out. The additional data included by specifying `--logtostderr` (or `--verbose 9`, e.g.) is just for diagnostic/debugging.